examples/Atlas/urdf/robotiq_remove_fourbar.urdf

```diff
@@ -0,0 +1,447 @@
+<?xml version="1.0" ?>
```

Comment:
@psiorx mentions that he removes the joint limits in `robotiq_simple.urdf` on purpose, so as to make it easier for LCP to solve the problem.